src/SassOptions.php

```diff
@@ -0,0 +1,135 @@
+<?php
+
+/*
+ * This file is part of the SymfonyCasts SassBundle package.
+ * Copyright (c) SymfonyCasts <https://symfonycasts.com/>
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfonycasts\SassBundle;
+
+/**
+ * --no-charset
+ * --style
+ * --no-source-map
+ * --embed-source-map.
+ *
+ * @see https://sass-lang.com/documentation/cli/dart-sass/
+ */
+final class SassOptions
```

Comment:
This class looks useless to me, especially as it is purely internal to the SassBuilder

Reply:
Removed